resources/views/items/datagrid.blade.php

```diff
@@ -12,6 +12,16 @@
         // Name
         'name',
         'type',
+        //Item_id
+        [
+            'label' => __('items.fields.item_id'),
+            'field' => 'item_id',
+            'render' => function($model) {
+                if ($model->item_id) {                    
```

Comment:
If you test just on `item_id`, it won't test the permission engine for non-admin viewers of a campaign. You always need to test `model->item`, because if the user doesn't have access to the item, it will return null.